src/front/wgsl.rs

```diff
@@ -970,6 +981,18 @@ impl Parser {
                             offset = lexer.next_uint_literal()?;
                             ready = false;
                         }
+                        Token::Word(word) if ready => {
```

Comment:
where are you getting the syntax? WGSL doesn't have any interpolation qualifiers defined yet
If you want to invent it, I'd much like to see the built-in decorator called "interpole", i.e. `[[interpolate linear]]` instead of just `[[linear]]`

Reply:
Yeah, I'm getting this syntax from [this issue](https://github.com/gpuweb/gpuweb/issues/802). Obviously, it's not official, but it seemed reasonable to me. This syntax here is easy enough to change when an official decision is reached, but I'd like to add it (or at least error out explicitly) here.

`[[interpolate <qualifier>]]` seems reasonable to me. I can change it to that if you'd like.